plugins/browse-pvc.yaml

```diff
@@ -0,0 +1,32 @@
+# Krew Manifest
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: browse-pvc
+spec:
+  version: "v0.3.0"
+  homepage: https://github.com/clbx/kubectl-browse-pvc
+  shortDescription: "Browse PVCs from the command line"
+  description: "Browse PVCs from the command line"
+  platforms:
+  # macOS
+  - selector:
+      matchLabels:
+        os: darwin
+    uri: https://github.com/clbx/kubectl-browse-pvc/releases/download/v0.3/kubectl-browse-pvc-darwin.zip
+    sha256: "0e0fc4e163e8434f9142edaa3b41fb1246e5c4dc652947284d6b8d49334601bd"
+    files:
```

Comment:
You can omit this field (as this value is the default)

Reply:
Removed